crates/bevy_sprite/src/tilemap_chunk/tilemap_chunk_material.wgsl

```diff
@@ -0,0 +1,58 @@
+#import bevy_sprite::{
+    mesh2d_functions as mesh_functions,
+    mesh2d_view_bindings::view,
+}
+
+struct Vertex {
+    @builtin(instance_index) instance_index: u32,
+    @location(0) position: vec3<f32>,
+    @location(1) uv: vec2<f32>,
+    @location(5) tile_index: u32,
```

Comment:
`tile_index` is unnecessary, can just be computed from `@builtin(vertex_index) v_id: u32,`